test/ranges.jl

```diff
@@ -717,7 +717,7 @@ end
 @testset "comparing and hashing ranges" begin
     Rs = AbstractRange[1:1, 1:1:1, 1:2, 1:1:2,
                        map(Int32,1:3:17), map(Int64,1:3:17), 1:0, 1:-1:0, 17:-3:0,
-                       0.0:0.1:1.0, map(Float32,0.0:0.1:1.0),
+                       0.0:0.1:1.0, map(Float32,0.0:0.1:1.0),map(Float32,LinRange(1.5, 5.5, 9)),
```

Comment:
This test is supposed to catch cases where arrays are equal and should hash equal. So better use a value which is equal to another range, like `LinRange(0.0, 1.0, 11)`.